src/jit/morph.cpp

```diff
@@ -4417,8 +4417,16 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* call)
 #else // !FEATURE_UNIX_AMD64_STRUCT_PASSING
 
 #ifndef LEGACY_BACKEND
-    // In the future we can migrate UNIX_AMD64 to use this
-    // method instead of fgMorphSystemVStructArgs
+// In the future we can migrate UNIX_AMD64 to use this
+// method instead of fgMorphSystemVStructArgs
+
```

Comment:
I think it looks more natural to keep the indentation here. To avoid formatting issue, you may use `CLANG_FORMAT_COMMENT_ANCHOR;`. It is often used when there is no C++ statements after comment lines.